parsely/src/main/java/com/parsely/parselyandroid/ParselyTrackerInternal.kt

```diff
@@ -316,7 +316,7 @@ internal class ParselyTrackerInternal internal constructor(
      * Any usage of this method is safe to remove and will have no effect. Keeping for backwards compatibility.
      */
     @Deprecated("The SDK now automatically flushes the queue on app lifecycle events. Any usage of this method is safe to remove and will have no effect")
-    override fun flushEventQueue() {
+    fun flushEventQueue() {
```

Comment:
I think I might have missed this in previous PR reviews. Since we already have a lot of breaking changes, should we simply remove this no-op function as well? It's probably OK either way, but I figure we might as well start fresh after all these breaking changes.

Reply:
Yes, definitely. I was considering doing this in a separate PR, but on the other hand, I can do this here. 59f21aa29ca3b5c05aa576017b442db7dd6f05bb